azure-pipelines.yml

```diff
@@ -13,38 +13,54 @@ strategy:
   matrix:
     linux_gcc6:
       VM_ImageName: 'ubuntu-16.04'
-      Compiler_ImageName: 'axom/compilers:gcc-6'
-      CMAKE_EXTRA_FLAGS: '-DAXOM_ENABLE_SIDRE=Off'
+      Compiler_ImageName: 'axom/tpls:gcc-6'
+      CMAKE_EXTRA_FLAGS: '-DAXOM_ENABLE_SIDRE=ON -DBUILD_SHARED_LIBS_ON'
```

Comment:
You don't have to explicitly set `AXOM_ENABLE_SIDRE=ON` here. It's `ON" by default. Also, there is a typo in `-DBUILD_SHARED_LIBS_ON`. I think the intent probably was `-DBUILD_SHARED_LIBS=ON`, but we shouldn't need this.